docs/install/pubsub-service-account.md

```diff
@@ -60,16 +60,37 @@ Workload Identity see
    during the control plane setup:
    [Install Knative-GCP](install-knative-gcp.md).
 
-   ```shell
-   gcloud services enable iamcredentials.googleapis.com
-   gcloud beta container clusters update ${CLUSTER_NAME} \
-   --identity-namespace=${PROJECT_ID}.svc.id.goog
-   ```
-
-1. Give `iam.serviceAccountAdmin` role to your control plane's Google Cloud
-   Service Account. You can skip this step if you granted `roles/owner`
-   privileges during the control plane setup:
-   [Install Knative-GCP](install-knative-gcp.md).
+    1. Ensure that you have enabled the Cloud IAM Service Account Credentials API.
```

Comment:
Instead of repeating the same steps, let's just reference the Workload Identity setup steps in install-knative-gcp.md.  I am doing so in the gcp broker setup guide: https://github.com/google/knative-gcp/pull/880/files#diff-ac3308c97418d861a1b386c0853c46d3R37